resources/kcp/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -42,6 +42,21 @@ spec:
       nodeSelector:
         {{- toYaml .Values.deployment.nodeSelector | nindent 8 }}
       containers:
+        {{- if .Values.broker.profiler.memory }}
+        - name: profiler
+          command:
+          - bash
+          - -c
+          - chmod 777 /tmp/profiler && sleep inf
+          securityContext:
+            runAsUser: 0
+          image: ubuntu:latest
```

Comment:
nit: should we rather use a specific version?

Reply:
good point, added that and rebased on latest main